tensorboard/util/test_util.py

```diff
@@ -209,10 +206,3 @@ def _impl(reason=None):
         return unittest.skipUnless(guard(), reason)
 
     return _impl
-
-
-run_v2_only = _run_conditionally(
```

Comment:
Might as well also remove _run_conditionally() since this is its last usage and I don't know that we have any immediately foreseeable need for it?

Reply:
Good point; thanks! Done.